src/main/kotlin/org/rust/cargo/runconfig/test/CargoTestEventsConverter.kt

```diff
@@ -0,0 +1,112 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.cargo.runconfig.test
+
+import com.google.gson.*
+import com.intellij.execution.testframework.TestConsoleProperties
+import com.intellij.execution.testframework.sm.ServiceMessageBuilder
+import com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter
+import com.intellij.openapi.util.Key
+import jetbrains.buildServer.messages.serviceMessages.*
+
+data class LibtestSuiteMessage(
+    val type: String,
+    val event: String
+) {
```

Comment:
Looks like that suite message can contain other useful info
For example, `test_count` in this event `{ "type": "suite", "event": "started", "test_count": "3" }`
Let's extract it and pass into test framework
See `com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter.MyServiceMessageVisitor.KEY_TESTS_COUNT` and `com.intellij.execution.testframework.sm.runner.OutputToGeneralTestEventsConverter.MyServiceMessageVisitor.ATTR_KEY_TEST_COUNT` constants